README.adoc

```diff
@@ -157,18 +157,18 @@ have IPv4 addresses, the address on eth2 is used.
 === HTTP Proxy Settings
 
 In an environment where the HTTP traffic needs to pass through an
-HTTP proxy server, set the `http_proxy`, `http_proxy_user` and
-`http_proxy_password` proxy configurations in the Vagrantfiles to enable
+HTTP proxy server, set the `proxy`, `proxy_user` and
```

Comment:
I don't think we can just change at will here. There was already a release with the _http.*_ variables. If we just change the names now we potentially existing users. So to do this properly, one needs to "deprecate" the old setting and for example print a warning when it is used, but still accept it. 

One also need to thing about introducing something like a migration guide which would document these type of changes between releases.